src/core/public/fatal_errors/fatal_errors_service.tsx

```diff
@@ -105,30 +95,26 @@ export class FatalErrorsService {
     return fatalErrorsSetup;
   }
 
-  private onFirstError() {
+  private onFirstError(injectedMetadata: InjectedMetadataSetup, i18n: I18nSetup) {
     // stop the core systems so that things like the legacy platform are stopped
     // and angular/react components are unmounted;
-    this.params.stopCoreSystem();
+    this.stopCoreSystem();
```

Comment:
optional: `command`-style from child to parent looks weird for me. I'd prefer `event`-style name. `onError`, for example